site/en/docs/privacy-sandbox/status/index.md

```diff
@@ -114,6 +118,20 @@ Descendant of [TURTLEDOVE](https://github.com/WICG/turtledove).
 - [GitHub](https://github.com/WICG/turtledove/blob/main/FLEDGE.md): see [TURTLEDOVE
   issues](https://github.com/WICG/turtledove/issues) for API questions and discussion.
 
+## Topics
+
+Replaces FLoC.
+
+- [Intent to
+  Experiment](https://groups.google.com/a/chromium.org/g/blink-dev/c/oTwd6VwCwqs).
+- [Intent to
+  Prototype](https://groups.google.com/a/chromium.org/g/blink-dev/c/59uTw_dxM3M/m/vF9lF9BVAgAJ).
+- [Blink status](https://groups.google.com/a/chromium.org/g/blink-dev/search?q="topics api").
```

Comment:
This appears to be broken
![Screen Shot 2022-03-31 at 8 13 39 AM](https://user-images.githubusercontent.com/2766281/161052181-b8f98d80-3b9f-463f-9e6c-c0cbb2ca01b8.png)

Reply:
@samdutton I'm not sure I understand why that's happening, the markdown is correct